tests/conftest.py

```diff
@@ -2,75 +2,91 @@
-    # "FruNet_model": "https://raw.githubusercontent.com/deepimagej/models/master/fru-net_sev_segmentation/model.yaml",
 }
 
+try:
+    import torch
+except ImportError:
+    torch = None
+skip_torch = torch is None
+
+try:
+    import onnxruntime
+except ImportError:
+    onnxruntime = None
+skip_onnx = onnxruntime is None
+
+try:
+    import tensorflow
+
+    tf_major_version = int(tensorflow.__version__.split(".")[0])
+except ImportError:
+    tensorflow = None
+    tf_major_version = None
+
+skip_tensorflow = tensorflow is None
+skip_tensorflow = True  # todo: update FruNet and remove this
+
+try:
+    import keras
+except ImportError:
+    keras = None
+skip_keras = keras is None
+skip_keras = True  # FruNet requires update
+
+# load all model packages we need for testing
+load_model_packages = set()
+if not skip_torch:
+    load_model_packages |= set(torch_models + torchscript_models)
+
+if not skip_onnx:
+    load_model_packages |= set(onnx_models)
+
+if not skip_tensorflow:
+    load_model_packages |= set(keras_models)
+    load_model_packages |= set(tensorflow_js_models)
+    if tf_major_version == 1:
+        load_model_packages |= set(tensorflow1_models)
+    elif tf_major_version == 2:
+        load_model_packages |= set(tensorflow2_models)
+
+
 # set 'skip_<FRAMEWORK>' flags as global pytest variables,
 # to deselect tests that require frameworks not available in current env
 def pytest_configure():
-    try:
-        import torch
-    except ImportError:
-        torch = None
-    pytest.skip_torch = torch is None
-
-    try:
-        import onnxruntime
-    except ImportError:
-        onnxruntime = None
-    pytest.skip_onnx = onnxruntime is None
-
-    try:
-        import tensorflow
-
-        pytest.tf_major_version = int(tensorflow.__version__.split(".")[0])
-    except ImportError:
-        tensorflow = None
-    pytest.skip_tensorflow = tensorflow is None
-    pytest.skip_tensorflow = True  # todo: update FruNet and remove this
-
-    try:
-        import keras
-    except ImportError:
-        keras = None
-    pytest.skip_keras = keras is None
-    pytest.skip_keras = True  # FruNet requires update
-
-    # load all model packages we need for testing
-    load_packages = {"unet2d_nuclei_broad_model"}  # always load unet2d_nuclei_broad_model
-    if not pytest.skip_torch:
-        load_packages |= set(torch_models + torchscript_models)
-
-    if not pytest.skip_onnx:
-        load_packages |= set(onnx_models)
-
-    if not pytest.skip_tensorflow:
-        load_packages |= set(keras_models)
-        load_packages |= set(tensorflow_js_models)
-        if pytest.tf_major_version == 1:
-            load_packages |= set(tensorflow1_models)
-        elif pytest.tf_major_version == 2:
-            load_packages |= set(tensorflow2_models)
-
-    pytest.model_packages = {name: export_resource_package(model_sources[name]) for name in load_packages}
+    pytest.skip_torch = skip_torch
```

Comment:
Are you still using the skip variables at all?

Reply:
now only `skip_onnx` is still in use due to some tests dependency on torch and onnx...